### ISSUE TITLE
paper issue

### ISSUE THREAD

**Issue by ggbox19990323** [GT-KIM/SELD1#2]
Could you please share the paper of this code? It doesn't seem like the paper in dcase 2022 Task 3 which your team provides.

**Comment by GT-KIM** [comment]
Since our paper was rejected at the ICASSP 2023, now we revise the manuscripts and implementations.
So, we will upload the paper after completing the revision.

Well, our team participated in DCASE 2022(Kim_KU entry), but the technical report(ConvNeXt ...) is NOT about this project.
This year, we also participate in DCASE 2023 task 3 based on this project. Therefore, the upgraded implementation and technical report will be publicly opened when the challenge ends.

**In summary, we will provide a new GitHub project and paper after DCASE 2023 challenge.**

Thank you for your interest in this project.

**Comment by ggbox19990323** [comment]
thank you very much
